Ninjathulhu/Ninjathulhu.Game.Test/PrefabTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace Ninjathulhu.Game.Test
+{
+    [TestFixture]
+    public class PrefabTests
+    {
+        private class SomeComponent : Component
```

Comment:
This needs to live somewhere, so I don't have to redefine it all over the damn place